File: Assignment_26/Assignment_26_1.java
import java.util.*;

class Pattern
{
    public void display(int iNo)
    {
       int iCnt  = 0;
       char ch = 'A';

        for(iCnt = 1; iCnt <= iNo; iCnt++,ch++)
        {
            System.out.printf("%c\n",ch);
            
        }
        System.out.println();
    }
}
class Assignment_26_1 
{
    public static void main(String A[])
    {
        int iValue = 0;

        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter Number");
        iValue = sobj.nextInt();


        Pattern pobj = new Pattern();
        pobj.display(iValue);

    }
    
}

File: Assignment_26/Assignment_26_1.c
#include<stdio.h>

void Display(int iNo)
{
    int iCnt  = 0;

        char ch = 'A';

        for(iCnt = 1; iCnt <= iNo; iCnt++,ch++)
        {
            printf("%c\n",ch);
            
        }
        printf("\n");
    
}

int main()
{
     int iValue = 0;

        
        printf("Enter Number :\n");
        scanf("%d",&iValue);

        Display(iValue);

    return 0;
}